indigo_app/templates/indigo_api/_task_card_single.html

```diff
@@ -57,6 +57,9 @@
         {% else %}
           #{{ task.id }}
         {% endif %}
+        {% if task.changes_requested %}
```

Comment:
A similar change should be added to the task list template.

Reply:
done, thanks!